src/libraries/System.Security.Cryptography.X509Certificates/tests/X509StoreMutableTests.Android.cs

```diff
@@ -0,0 +1,79 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.Security.Cryptography.X509Certificates.Tests
+{
+    [PlatformSpecific(TestPlatforms.Android)]
+    public static partial class X509StoreMutableTests
+    {
+        public static bool PermissionsAllowStoreWrite { get; } = true;
+
+        [Fact]
+        public static void AddRemove_Cert()
```

Comment:
This test looks just like VerifyRemove from the also-included partial file.  What's the difference?

Reply:
I was going for one that just does a plain cert. I could collapse this into VerifyRemove and make that have a cert-only version? (Or remove entirely if the plain cert test is not interesting to separate out)

Reply:
Ah.  Let's move this concept into the shared file and do one of

* Make the existing test do both things
* Make the existing test a theory with a `bool withPrivateKey`
* Make two tests, VerifyRemove_CertWithKey, VerifyRemove_CertOnly (or similar naming)